filters/ThirdParty/filter_250_DandelionSproutAnnoyances/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "filterId": 250,
+  "name": "Dandelion Sprout's Annoyances List",
+  "description": "This list is made in the style of Fanboy's and AdGuard's annoyances lists, to show that I can make a just as good such annoyances list as them. It combines many of my proudest and most frequently maintained international lists, as a proof-of-concept and as a curated list for simplicity's sake.",
```

Comment:
Btw, there is a new description https://github.com/AdguardTeam/FiltersRegistry/issues/400#issuecomment-792999470 without this part `, to show that I can make a just as good such annoyances list as them.`.
```
"This list is made in the style of AdGuard's and Fanboy's annoyances lists. It combines many of Dandelion Sprout's proudest and most frequently maintained international lists, as a curated compilation for simplicity's sake."
```

Reply:
Changed description